.env_example

```diff
@@ -0,0 +1,12 @@
+# Required
```

Comment:
do we want to provide sample values for these?

Reply:
Also might be good to provide some comments around what each defaults to w/o having to go look int `config.ts`

Reply:
I would provide sample values if it weren't for the `ETHEREUM_RPC_URL` value. Are we ok with putting a valid value in there?

The defaults are mentioned in the README.